app/cruds/cruds_recommendation.py

```diff
@@ -0,0 +1,83 @@
+import uuid
+from datetime import datetime
+from typing import Sequence
+from zoneinfo import ZoneInfo
+
+from sqlalchemy import delete, select, update
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.core.config import Settings
+from app.models import models_recommendation
+from app.schemas import schemas_recommendation
+
+
+async def get_recommendation(
+    db: AsyncSession,
+) -> Sequence[models_recommendation.Recommendation]:
+    result = await db.execute(select(models_recommendation.Recommendation))
+    return result.scalars().all()
+
+
+async def create_recommendation(
+    recommendation: schemas_recommendation.RecommendationBase,
```

Comment:
Could we directly ask for a model here?

Reply:
Yes but what would be the advantage of creating the recommendation database object inside the endpoint rather than in the crud. As this is related to the database, I thought that it made more sense to define it inside the crud rather than in the endpoint.